src/resources/js/vue/components/tables/TableRequests.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="row no-gutters h-100 justify-content-center align-items-center">
+    <table class="table table-striped table-bordered table-hover">
+      <thead class="table-header">
+        <tr>
+          <th>Origen</th>
+          <th>Destino</th>
+          <th>Fecha</th>
+          <th>N° Ofertas</th>
+          <th>Estado</th>
+          <th class="w-25">
+            Acciones
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="(request, index) in requests"
+          :key="index">
+          <td>{{ request.origin }}</td>
+          <td>{{ request.destination }}</td>
+          <td>{{ request.date }}</td>
+          <td>{{ request.offers }}</td>
+          <td>{{ request.state }}</td>
+          <td>
+            <button
+              class="btn btn-link"
+              :disabled="request.state !== 'pendiente'"
+              type="button">
+              <font-awesome-icon
+                class="navbar-icon"
+                icon="edit"
+                size="1x"/>
+              Editar
+            </button>
+            <button
+              class="btn btn-link"
+              :class="{ 'btn-delete' : request.state === 'realizado' || request.state === 'cancelado' }"
```

Comment:
We can put all the status in a constants file
Let say constants/requests and inside it all the posible status